doc/versionHistory.rst

```diff
@@ -6,6 +6,12 @@
 Version History
 ##################
 
+-------------
```

Comment:
Above that it still says `lsst.ts.wep`  and `py:currentmodule:: lsst.ts.wep`  - would you like to correct it  in this PR ?

Reply:
Oh wow, good catch. Yeah, let's fix that. Thanks.